frontend-ui/src/components/FileInfoTable.vue

```diff
@@ -0,0 +1,154 @@
+        <v-divider v-if="fileInfo.media_count !== undefined" class="my-1" />
+        <div v-if="fileInfo.size !== undefined" class="py-1">
+          <v-row dense class="align-start">
+            <v-col cols="4" class="text-body-2 font-weight-medium">Size</v-col>
+            <v-col cols="8" class="text-body-2">{{ formattedBytesSize(fileInfo.size) }}</v-col>
+          </v-row>
+        </div>
+        <v-divider v-if="fileInfo.size !== undefined" class="my-1" />
+
+        <!-- Counter entries -->
+        <div v-if="Object.keys(fileInfo.counter || {}).length > 0" class="py-2">
+          <v-row dense class="align-start">
+            <v-col cols="4" class="text-body-2 font-weight-medium">Counter</v-col>
+            <v-col cols="8">
+              <div class="bg-grey-lighten-5 rounded pa-2">
+                <v-row dense class="text-caption font-weight-medium mb-1">
+                  <v-col cols="9">MIME</v-col>
+                  <v-col cols="3" class="text-right">Count</v-col>
+                </v-row>
+                <v-divider class="mb-2" />
+                <div
+                  v-for="(value, key, index) in fileInfo.counter"
+                  :key="key"
+                  :class="index % 2 === 0 ? 'bg-grey-lighten-3' : ''"
+                >
+                  <v-row dense class="py-1 px-2">
+                    <v-col cols="9" class="text-body-2 text-break">{{ key }}</v-col>
+                    <v-col cols="3" class="text-body-2 text-right">{{
+                      value.toLocaleString()
+                    }}</v-col>
+                  </v-row>
+                </div>
+              </div>
+            </v-col>
+          </v-row>
+        </div>
+        <v-divider v-if="Object.keys(fileInfo.counter || {}).length > 0" class="my-2" />
+
+        <!-- Metadata entries -->
+        <div v-if="filteredMetadataEntries.length > 0" class="py-2">
+          <v-row dense class="align-start">
+            <v-col cols="4" class="text-body-2 font-weight-medium">Metadata</v-col>
+            <v-col cols="8">
+              <div class="bg-grey-lighten-5 rounded pa-2">
+                <div
+                  v-for="([key, value], index) in filteredMetadataEntries"
+                  :key="key"
+                  :class="index % 2 === 0 ? 'bg-grey-lighten-3' : ''"
+                >
+                  <v-row dense class="align-start py-1 px-2">
+                    <v-col cols="4" md="3" class="text-body-2 font-weight-medium">{{ key }}</v-col>
+                    <v-col cols="8" md="9" class="text-body-2 text-break">
+                      <!-- Handle illustration_ attributes as base64 images -->
+                      <div
+                        v-if="isIllustrationAttribute(key)"
+                        class="d-flex flex-column align-start"
+                      >
+                        <img
+                          :src="`data:image/png;base64,${value}`"
+                          :alt="key"
+                          style="
```

Comment:
We need the illustration to be exactly original illustration:
- if metadata key is `Illustration_48x48`, then image in info box must be 48x48, and so on (we only have 48x48 illustrations for now, but we do plan to add more in the future)
- no border, no radius
- `object-fit: contain` is probably correct since this is what most readers would do (image should already be 48x48 anyway)